site/v5/professional/professional-setup.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Set up
+date: 2014-04-18 11:30:00 
+permalink: /v5/professional/setup.html
+eleventyNavigation:
+  version: v5
+  order: 1
+---
+<h1 class="title">Professional - Set up</h1>
+
+[[TOC]]
+
+# Introduction
+This sections explains and goes through all the steps to get up and running the Mongock professional.
+
+
+# Steps
+
+### 1. Request a license key
+
+You can request a License key from the following [link](https://www.mongock.io/download). The website will display a form which will send a license key to the desired email address.
+
+### 2. Chage the groupId in you pom
+
+If you are using Mongock Community edition with Maven, your POM file should have a dependency with the following groupId:
```

Comment:
I think we should clarify here that the Mongock's dependency to be modified is runner's dependency. In your example you put "springboot" dependency, but no explained it. It might be "standalone" instead.